ReactNativeClient/lib/SyncTargetOneDrive.js

```diff
@@ -71,6 +71,14 @@ class SyncTargetOneDrive extends BaseSyncTarget {
 	}
 
 	async initFileApi() {
+		let acctProps = Setting.value(`sync.${this.syncTargetId()}.accountProperties`);
+		if (acctProps === '') {
+			acctProps = await this.api_.execAcctPropsRequest();
+			Setting.setValue(`sync.${this.syncTargetId()}.accountProperties`, JSON.stringify(acctProps));
+		} else {
+			acctProps = JSON.parse(acctProps);
+		}
+		this.api_.setAcctProps(acctProps);
```

Comment:
Please use full words in general, as not everybody will know that "acct" means "account". So "accountProperties", "setAccountProperties"